bin/fin

```diff
@@ -2514,8 +2515,20 @@ project_create ()
 				target_host_name_search_string=""
 				target_repo="$URL_REPO_LARAVEL"
 				;;
+	        9)
+				target_host_name_search_string=""
+				target_cms="Custom git repository"
+				while true; do
+					read -p "Please enter repository url: " target_repo
+					if [[ -z "${target_repo// }" ]]; then
+						echo-red "Please provide repository url."
```

Comment:
Wouldn't this output 'Please provide repository url' twice if you didn't provide a value?